agent/consul/server.go

```diff
@@ -139,20 +138,8 @@ type Server struct {
 	// autopilot is the Autopilot instance for this server.
 	autopilot *autopilot.Autopilot
 
-	// autopilotWaitGroup is used to block until Autopilot shuts down.
-	autopilotWaitGroup sync.WaitGroup
```

Comment:
should this have been deleted?

Reply:
It wasn't referenced by anything else, so I thought it had already been removed